### DISCUSSION TITLE
Error "Command "npm run build" exited with 1" while trying to deploy

### DISCUSSION THREAD

**Discussion by dianas2000** [vercel/community#5018] [Help]
### Summary

Hi everyone! 
I edited my code and wanted it to publish it but instead I am getting this error:
<img width="1346" alt="Bildschirmfoto 2023-11-29 um 16 10 43" src="https://github.com/vercel/community/assets/148540281/b0b2b4b1-fc6c-439d-b3d0-1c21a98573f7">

### Steps to Reproduce

What can be the problem? Please help since I need to have the website ready until tomorrow!

**Comment by dianas2000** [comment]
dianas2000/portfolio is my Code

**Comment by ItzCrazyKns** [comment]
This issue is generally caused by warnings that are being taken as errors when building. You should check the logs and fix them or if you want to forcefully deploy the project you can set the `CI` enviroment variable in your project settings to `false`.

**Reply by dianas2000** [reply]
Thank you for your response. I have already done that and nothing has changed :(

**Reply by ItzCrazyKns** [reply]
Can you check the logs and see what's there? Also try to change the build command to `npm run build`

**Reply by dianas2000** [reply]
<img width="1001" alt="Bildschirmfoto 2023-11-29 um 16 37 34" src="https://github.com/vercel/community/assets/148540281/9d8b0490-7289-4bbb-a1d9-5c3a04a41174">

I also changed the command

**Reply by ItzCrazyKns** [reply]
Not this log, the build log
